include/seahorn/UfoOpMemSem.hh

```diff
@@ -0,0 +1,83 @@
+#pragma once
```

Comment:
Why is `UfoOpMemSem` has been pulled into a header file?

Reply:
There were originally an `UfoOpSem.hh` (and `UfoOpSem.cc`), so I used the same structure in my copy. Should I move `UfoOpMemSem.hh` to `UfoOpSem.hh`?

Reply:
No. This is good. Just add a comment that this is a clone of `UfoOpSem` for the purpose of prototyping. I got confused and thought this was an existing class.

Reply:
OK!